src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageUpdaterStep.cs

```diff
@@ -155,7 +156,7 @@ protected override async Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeCon
                         count++;
 
                         Logger.LogDebug("Re-running analysis to check whether additional changes are needed");
-                        _analysisState = await _packageAnalyzer.AnalyzeAsync(context, context.CurrentProject, token).ConfigureAwait(false);
+                        _analysisState = await _packageAnalyzer.AnalyzeAsync(context, project, project.TargetFrameworks, token).ConfigureAwait(false);
```

Comment:
Should this be passing in the selected TFM for the project?

Reply:
This is the same code that is being called for Pre-TFM step as well, so if we change this to use target TFM always then the package dependencies will be updated for target TFM before the TFM is updated, which is Ok I guess. We would then need to remove the Pre-TFM package update step probably.

Reply:
That would probably be a breaking change here, coz this is applied during pre-TFM change too.

Reply:
FYI @mjrousos

Reply:
This is probably correct for now because it will preserve the tool's current behavior (we won't look for new Net5 versions of package the first time the step runs but will the second time (after updating the TFM)).

I suspect once we have a solution-wide package update step, we'll need to pass the new selected TFM in those cases but we can worry about that later.

Reply:
leaving it as is for now